docs/sources/setup-grafana/configure-security/configure-authentication/saml-ui/index.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Learn how to configure SAML authentication in Grafana's UI.
+menuTitle: Configure SAML using the UI
+title: Configure SAML authentication using the Grafana user interface
+weight: 1150
+---
+
+# Configure SAML authentication using the Grafana user interface
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../introduction/grafana-enterprise/" >}}) version 10.0 and later, and [Grafana Cloud Pro and Advanced](/docs/grafana-cloud/).
+
+You can configure SAML authentication in Grafana through the user interface (UI) or the Grafana configuration file. For instructions on how to set up SAML using the Grafana configuration file, refer to [Configure SAML authentication using the configuration file]({{< relref "../saml/" >}}).
+
+The Grafana SAML UI provides the following advantages over configuring SAML in the Grafana configuration file:
+
+- It is accessible by hosted Grafana users
+- It doesn't require Grafana to be restarted after a configuration update
+- Access to the SAML UI only requires access to authentication settings, so it can be used by users with limited access to Grafana's configuration
+
+> **Note:** Configuration in the UI takes precedence over the configuration in the Grafana configuration file. SAML settings from the UI will override any SAML configuration set in the Grafana configuration file.
+
+## Before you begin
+
+To follow this guide, you need:
+
+- Knowledge of SAML authentication. Refer to [SAML authentication in Grafana]({{< relref "../saml/" >}}) for an overview of Grafana's SAML integration.
+- Permissions that allow you to read and update authentication settings. These permissions are granted by `fixed:authentication.config:writer` role.
```

Comment:
I could not find this role in the [definitions](https://grafana.com/docs/grafana/next/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/). Should we add this in scope of this PR, or separate? Also what do you think about mentioning the permission ID itself, next to the role?

Reply:
I raised a separate small PR for it: https://github.com/grafana/grafana/pull/66957